resources/views/login.blade.php

```diff
@@ -7,15 +7,32 @@
 
 @section('classes_body', 'login-page')
 
+@php( $login_url = View::getSection('login_url') ?? config('adminlte.login_url', 'login') )
+@php( $register_url = View::getSection('register_url') ?? config('adminlte.register_url', 'register') )
+@php( $password_reset_url = View::getSection('password_reset_url') ?? config('adminlte.password_reset_url', 'password/reset') )
+@php( $dashboard_url = View::getSection('dashboard_url') ?? config('adminlte.dashboard_url', 'home') )
+
+@if (config('adminlte.use_route_url', false))
+    @php( $login_url = $login_url ?? route($login_url) )
+    @php( $register_url = $register_url ?? route($register_url) )
+    @php( $password_reset_url = $password_reset_url ?? route($password_reset_url) )
+    @php( $dashboard_url = $dashboard_url ?? route($dashboard_url) )
+@else
+    @php( $login_url = $login_url ?? url($login_url) )
+    @php( $register_url = $register_url ?? url($register_url) )
+    @php( $password_reset_url = $password_reset_url ?? url($password_reset_url) )
+    @php( $dashboard_url = $dashboard_url ?? url($dashboard_url) )
+@endif
```

Comment:
There may be some mistakes when using Null Coalescing Operator `??`. The second expression of this operator usually be a default value.

[Null Coalescing Operator - PHP: Comparison Operators - Manual](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce)

Do you means?

```php
$login_url = route($login_url ?? 'login');
```
Or maybe

```php
$login_url = $login_url ? route($login_url) : '';
```

Reply:
Oh dang you are right, I'll fix this soon in a PR 😄 It should look like your second example.

The strangest part is it works as it should, including disabled register link.